cis/Issues/official/description.xml

```diff
@@ -0,0 +1,33 @@
+<IssueSpecification version="4">
+	<Name>
+		<en>Demo Stichting Centraal Informatie Systeem</en>
+		<nl>Demo Stichting Centraal Informatie Systeem</nl>
+	</Name>
+	<ShortName>
+		<en>Demo CIS</en>
+		<nl>Demo CIS</nl>
+	</ShortName>
+	<SchemeManager>irma-demo</SchemeManager>
+	<IssuerID>cis</IssuerID>
+	<CredentialID>official</CredentialID>
```

Comment:
Maybe it's nice to use a bit more descriptive credential ID. If you'd ever add a second credential, then it is easier to distinguish both in your back-end.

Reply:
changed credentialID to a more descriptive name